cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,54 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import PageObject from "../support/PageObject";
+import HomeAndCataloguePageObject from "../support/pages/homeСatalogue.pageObject";
+import { ProductPage } from "../support/pages/product.pageObject";
+import { CartPage } from "../support/pages/cart.pageObject";
+
+const homePage = new HomeAndCataloguePageObject();
+const pageObject = new PageObject();
+const productPage = new ProductPage()
+const cartPage = new CartPage();
+const faker = require("faker");
+
+const infoAboutUser = {
+  name: faker.name.firstName() + faker.name.lastName(),
+  country: faker.address.country(),
+  city: faker.address.city(),
+  creditCard: '9876123456789456',
+  month: faker.date.month(),
+  year: '1990'
+}
 
+describe('Demoblaze', () => {
+  before(() => {
+    homePage.visit();
   });
 
-  it('', () => {
+  it('should provide an ability to add item to the cart and buy it', () => {
+    homePage.clickOnCategory('Laptops');
+    homePage.clickOnProduct('Sony vaio i7');
+    productPage.clickOnAddToCartBtn();
+    cy.wait(1000);
+
+    pageObject.assertAllert('Product added');
+
+    homePage.clickOnLink('Cart');
+
+    cartPage.assertProductInCart('Sony vaio i7');
+
+    cartPage.clickOnPlaceOrderBtn();
+    cartPage.getById('name', infoAboutUser.name);
```

Comment:
I think method getById should return something. But here I see that you fill the fields/
```suggestion
    cartPage.fillTheField('name', infoAboutUser.name);
```

Reply:
create separate methods for every field and use there fillTheField function